xbmc/powermanagement/PowerManager.cpp

```diff
@@ -34,9 +35,17 @@
 extern HWND g_hWnd;
 #endif
 
-using namespace ANNOUNCEMENT;
-
-CPowerManager::CPowerManager() = default;
+CPowerManager::CPowerManager(ANNOUNCEMENT::CAnnouncementManager &announcements,
+                             CSettings &settings,
+                             CNetworkBase &network,
+                             PVR::CPVRManager &pvr) :
+  m_announcements(announcements),
+  m_settings(settings),
+  m_network(network),
+  m_pvr(pvr)
```

Comment:
PVR has ever been an optional component. Creating this hard dependency is not a good idea.